tests/test_plugin.py

```diff
@@ -27,3 +29,25 @@ def test_option_passthrough(node_factory):
     # option didn't exist
     n = node_factory.get_node(options={'plugin': plugin_path, 'greeting': 'Mars'})
     n.stop()
+
+
+def test_rpc_passthrough(node_factory):
+    """Starting with a plugin exposes its RPC methods.
+
+    First check that the RPC method appears in the help output and
+    then try to call it.
+
+    """
+    plugin_path = 'contrib/plugins/helloworld.py'
+    n = node_factory.get_node(options={'plugin': plugin_path, 'greeting': 'Mars'})
+
+    # Make sure that the 'hello' command that the helloworld.py plugin
+    # has registered is available.
+    cmd = [hlp for hlp in n.rpc.help()['help'] if 'hello' in hlp['command']]
+    assert(len(cmd) == 1)
+
+    # Now try to call it and see what it returns:
+    greet = n.rpc.hello(name='Sun')
```

Comment:
meta: how will developers that use the python RPC interface know that there is a new RPC-call available? Even if plugin developers would inharite from the LightningRPC class this would not help as this derived class is not shipped via pip3. 

I believe the best way to handle this is to change the pylightning.py file in a way so that the available RPC calls provide methods dynamically (maybe via reflection?)

Reply:
Unknown rpc calls are handled in pylightning, but it's a bit obscure IMHO. You need to call it using a dict of options, not inline, IIRC....

Reply:
We can certainly make pylightning a bit smarter to accept either all list arguments or all keyword arguments. I'll note that for the next cleanup PR.